Model/AbstractApi.php

```diff
@@ -188,10 +227,23 @@ public function oauthAuthentication($scopeId = null, $scope = null)
                 $this->_yotpoConfig->log("AbstractApi::oauthAuthentication({$scopeId}, {$scope}) - error: no access token received", "error", ['$app_key' => $app_key]);
                 return null;
             }
+            $this->resourceConfig->saveConfig(YotpoConfig::XML_PATH_YOTPO_TOKEN, $this->encryptor->encrypt($token), 'stores', $scopeId);
```

Comment:
why are you deleting all stores tokens?

Reply:
Actually, we are not deleting all the stores token only the corresponding store token which will be in the processing state. 
Anyways, this is not required any more as per the changes we are doing in this comment "**I think that instead of accessing yotpo config you can just force create a token here no?
$this->oauthAuthentication($this->_yotpoConfig->getCurrentStoreId(), true)**"

Many thanks